tests/e2e-test-framework/framework/ssh.py

```diff
@@ -1,51 +1,19 @@
 from typing import Any, List, Tuple
-
 import logging
 import paramiko
 
 
-class SSHCommandExecutor:
```

Comment:
issue:  It shouldn't have been removed. It's a core class used by many other utils.